primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -144,68 +144,68 @@ if (window.PrimeFaces) {
+      "javax.faces.converter.BooleanConverter.BOOLEAN_detail": "{1}: '{0}' debe ser 'verdadero' o 'falso'.  Cualquier valor diferente a 'verdadero' se evaluará como 'falso'.",
+      "javax.faces.validator.LongRangeValidator.MAXIMUM": "{1}: Error de validación: el valor es superior al máximo permitido de '{0}'",
+      "javax.faces.validator.LongRangeValidator.MINIMUM": "{1}: Error de validación: el valor es inferior al mínimo permitido de '{0}'",
       "javax.faces.validator.LongRangeValidator.NOT_IN_RANGE": "{2}: Error de validación: el atributo especificado no está entre los valores esperados {0} y {1}.",
       "javax.faces.validator.LongRangeValidator.TYPE": "{0}: Error de validación: el valor no tiene el tipo correcto.",
-      "javax.faces.validator.DoubleRangeValidator.MAXIMUM": "{1}: Error de validación: el valor es mayor que el máximo permitido de '{0}'",
-      "javax.faces.validator.DoubleRangeValidator.MINIMUM": "{1}: Error de validación: el valor es inferior que el mínimo permitido de '{0}'",
+      "javax.faces.validator.DoubleRangeValidator.MAXIMUM": "{1}: Error de validación: el valor es superior al máximo permitido de '{0}'",
+      "javax.faces.validator.DoubleRangeValidator.MINIMUM": "{1}: Error de validación: el valor es inferior al mínimo permitido de '{0}'",
       "javax.faces.validator.DoubleRangeValidator.NOT_IN_RANGE": "{2}: Error de validación: el atributo especificado no está entre los valores esperados {0} y {1}",
       "javax.faces.validator.DoubleRangeValidator.TYPE": "{0}: Error de validación: el valor no tiene el tipo correcto.",
-      "javax.faces.converter.FloatConverter.FLOAT": "{2}: '{0}' debe ser un número formado por uno o varios dígitos.",
+      "javax.faces.converter.FloatConverter.FLOAT": "{2}: '{0}' debe ser un número formado por uno o más dígitos.",
       "javax.faces.converter.FloatConverter.FLOAT_detail": "{2}: '{0}' debe ser un número entre 1.4E-45 y 3.4028235E38  Ejemplo: {1}",
       "javax.faces.converter.DateTimeConverter.DATE": "{2}: '{0}' no se ha podido reconocer como fecha.",
       "javax.faces.converter.DateTimeConverter.DATE_detail": "{2}: '{0}' no se ha podido reconocer como fecha. Ejemplo: {1}",
       "javax.faces.converter.DateTimeConverter.TIME": "{2}: '{0}' no se ha podido reconocer como hora.",
       "javax.faces.converter.DateTimeConverter.TIME_detail": "{2}: '{0}' no se ha podido reconocer como hora. Ejemplo: {1}",
       "javax.faces.converter.DateTimeConverter.DATETIME": "{2}: '{0}' no se ha podido reconocer como fecha y hora.",
       "javax.faces.converter.DateTimeConverter.DATETIME_detail": "{2}: '{0}' no se ha podido reconocer como fecha y hora. Ejemplo: {1}",
-      "javax.faces.converter.DateTimeConverter.PATTERN_TYPE": "{1}: debe especificarse el atributo 'pattern' o 'type' para convertir el valor '{0}'",
+      "javax.faces.converter.DateTimeConverter.PATTERN_TYPE": "{1}: debe especificarse el atributo 'patrón' o 'tipo' para convertir el valor '{0}'",
```

Comment:
I'm not sure in this case but I think the words "patrón"y "tipo" should be in English.

Reply:
Yes, I've have doubts too. Maybe, not need to change.